src/mcts/params.cc

```diff
@@ -104,6 +104,12 @@ const OptionId SearchParams::kSmartPruningFactorId{
     "promising moves from being considered even earlier. Values less than 1 "
     "causes hopeless moves to still have some attention. When set to 0, smart "
     "pruning is deactivated."};
+const OptionId SearchParams::kEarlyRootWideningId{
```

Comment:
Maybe have pair of params `FpuStrategyAtRoot` (with values `absolute`, `relative` and `same` for same is not in root), and `FpuValueAtRoot`

Reply:
Updated PR as suggested